packages/grid/x-data-grid/src/components/cell/GridEditSingleSelectCell.tsx

```diff
@@ -42,6 +28,12 @@ export interface GridEditSingleSelectCellProps
    * If true, the select opens by default.
    */
   initialOpen?: boolean;
+  /**
+   * Used to determine the text displayed for a given value option.
+   * @param {ValueOptions} value The current value option.
+   * @returns {string} The text to be displayed.
+   */
+  getOptionLabel?: (value: ValueOptions) => React.ReactNode;
```

Comment:
Shouldn't it be declared in `gridColDef.ts`?
Also, should it return a string?
```suggestion
  getOptionLabel?: (value: ValueOptions) => string;
```